.github/CODEOWNERS

```
@@ -0,0 +1,3 @@
+# See https://help.github.com/en/articles/about-code-owners
+
+* @corycaywood @donnie-jp @ts-nitish-a-jain @minh-rakuten @ts-leojoseph-r @Climbatize
```

Looks okay. Alphabetical order is easier to read 👍 
```suggestion
* @Climbatize @corycaywood @donnie-jp @minh-rakuten @ts-leojoseph-r @ts-nitish-a-jain 
```